app-admin/syslog-ng/files/3.15/syslog-ng.conf.gentoo

```diff
@@ -0,0 +1,36 @@
+@version: 3.15
```

Comment:
Could you look into not duplicating those files for every major release? I suppose you could sed the `@version` in if it's really necessary.